app/overrides/decorate_admin_configurations_index.rb

```diff
@@ -1,5 +1,11 @@
-Deface::Override.new(:virtual_path => "spree/admin/shared/_configuration_menu",
-                     :name => "multi_domain_admin_configurations_menu",
-                     :insert_bottom => "[data-hook='admin_configurations_sidebar_menu']",
-                     :text => "<%= configurations_sidebar_menu_item Spree.t(:stores_admin), admin_stores_url %>",
-                     :disabled => false)
+Deface::Override.new(
+  virtual_path: "spree/admin/shared/_configuration_menu",
+  name: "multi_domain_admin_configurations_menu",
+  insert_bottom: "[data-hook='admin_configurations_sidebar_menu']",
+  disabled: false,
+  text: <<-EOF
+  <% if can?(:display, Spree::Store) %>
+    <%= configurations_sidebar_menu_item Spree.t(:stores_admin), admin_stores_url %>
+  <% end %>
+  EOF
```

Comment:
Can you extract this to a template?

It makes it easier to override if you want to change this.